### PR TITLE
refactor: replace all master to main

### PR DESCRIPTION
**Pull request by hong4rc**
Currently, we implicitly `master` is `main`.

Use **real** main is better.

#821

I replace:

 - `\bmaster\b`` -> `main`

- `\bMaster\b` -> `Main`

- `compareOnlyMaster` -> `compareOnlyMain`

- `Master\b` -> `Main`

- `%22master\b` -> `%22main`

- remove `.replace(/\bmaster\b/, 'main')`

- `master\b` -> `main`

- `\bmaster([ぁ-んァ-ン])` -> `main$1`

- `([ぁ-んァ-ン])master([ぁ-んァ-ン])` -> `$1main$2`

- `([\u4E00-\u9FCC]|\b)master([\u4E00-\u9FCC]|\b)` -> `$1main$2`

- `MaStEr` -> `MaIn, mAsTeR` -> `mAiN`

- `\b([\w\d]+)Master([\w\d]+)\b` -> `$1Main$2`

- `\bmaster([A-Z][\w\d]*)\b` -> `main$1`

- file `remote/lockedMaster` -> `remote/lockedMain`

image
Idk how to change this `master`

### CODE REVIEW

**Comment by pcottle** [comment]
oh this is one spot we should not change over, since the repo itself still operates on master:

```
  execSync('git checkout gh-pages && git merge main -m "merge main"');
```
(in the gulp file)

**Comment by pcottle** [comment]
Pushed the site! looks fine so far

File: src/js/dialogs/sandbox.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ exports.dialog = {
         'Try out ',
         '[this special link](https://pcottle.github.io/learnGitBranching/?NODEMO)',
         '',
-        'PPS: GitHub (and the industry at large) is moving to naming the default branch as `main` instead of `master` ',
+        'PPS: GitHub (and the industry at large) is moving to naming the default branch as `main` instead of `main` ',
```
Comment (waldyrious) [inline src/js/dialogs/sandbox.js:23]: @pcottle @hong4rc this one also should not have been changed! 😄

Comment (pcottle) [reply src/js/dialogs/sandbox.js:23]: Ah yeah good catch -- will update

Comment (pcottle) [reply src/js/dialogs/sandbox.js:23]: https://github.com/pcottle/learnGitBranching/commit/ab3a2349edb62e8d396b64930aed59972e2486a2

Comment (pcottle) [reply src/js/dialogs/sandbox.js:23]: Screen Shot 2021-04-19 at 1 14 35 PM

**Comment by pcottle** [comment]
This also inspired me to rename the branch in the repo!

```
remote: 
To github.com:pcottle/learnGitBranching.git
   2df860e..a9b07dd  main -> main
```

**Comment by pcottle** [comment]
Updated the script too:
https://github.com/pcottle/learnGitBranching/commit/a9b07dd95ee2bfc3cf90affb0e632707ea836f24

**Comment by ElLutzo** [comment]
Ah sorry, did not look into this merge discussion.
But should have surfaced in the application immediately (at least for en_US) if not in this review.
Either way: done. ☑️, thanks.

**Pull request merged**